__init__.py

```diff
@@ -1 +1,85 @@
+import platform
+import sys
+import os
+import subprocess
+import torch
+
+# Python version and platform checks
+python_version = f"{sys.version_info.major}.{sys.version_info.minor}"
+platform_system = platform.system()
+platform_machine = platform.machine()
+
+print(f"Detected Python version: {python_version}")
+print(f"Detected platform system: {platform_system}")
+print(f"Detected platform machine: {platform_machine}")
+
+# List of dependencies
+requirements = []
+
+# llama-cpp-python (CPU only, AVX2)
+cpu_wheels = {
+    "3.11": {
+        "Linux": {
+            "x86_64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/cpu/llama_cpp_python-0.2.89+cpuavx2-cp311-cp311-linux_x86_64.whl"
+        },
+        "Windows": {
+            "amd64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/cpu/llama_cpp_python-0.2.89+cpuavx2-cp311-cp311-win_amd64.whl"
+        }
+    },
+    "3.10": {
+        "Linux": {
+            "x86_64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/cpu/llama_cpp_python-0.2.89+cpuavx2-cp310-cp310-linux_x86_64.whl"
+        },
+        "Windows": {
+            "amd64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/cpu/llama_cpp_python-0.2.89+cpuavx2-cp310-cp310-win_amd64.whl"
+        }
+    }
+}
+
+# llama-cpp-python (CUDA, no tensor cores)
+cuda_wheels = {
+    "3.11": {
+        "Linux": {
+            "x86_64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/textgen-webui/llama_cpp_python_cuda-0.2.89+cu121-cp311-cp311-linux_x86_64.whl"
+        },
+        "Windows": {
+            "amd64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/textgen-webui/llama_cpp_python_cuda-0.2.89+cu121-cp311-cp311-win_amd64.whl"
+        }
+    },
+    "3.10": {
+        "Linux": {
+            "x86_64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/textgen-webui/llama_cpp_python_cuda-0.2.89+cu121-cp310-cp310-linux_x86_64.whl"
+        },
+        "Windows": {
+            "amd64": "https://github.com/oobabooga/llama-cpp-python-cuBLAS-wheels/releases/download/textgen-webui/llama_cpp_python_cuda-0.2.89+cu121-cp310-cp310-win_amd64.whl"
+        }
+    }
+}
+
+# Function to install a package
+def install_package(package):
+    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
+
+# Installing base requirements
+for req in requirements:
```

Comment:
This will always install these packages, even if they are already installed on the system.